app/views/users/_profile_actions.html.erb

```diff
@@ -1,10 +1,12 @@
-    <!-- profile and view buttons -->
-      <% if signed_in? %>
-        <% if current_user == @user %>
-          <%= link_to raw('<i class="glyphicon glyphicon-edit"></i> Edit Your Profile'), sufia.edit_profile_path(@user), class: "btn pull-left subtab" %>
-        <% elsif current_user.following?(@user) %>
-          <%= link_to raw('<i class="glyphicon glyphicon-eye-close"></i> Unfollow'), sufia.unfollow_profile_path(@user), method: :post, class: "btn pull-left subtab" %>
-        <% else %>
-          <%= link_to raw('<i class="glyphicon glyphicon-eye-open"></i> Follow'), sufia.follow_profile_path(@user), method: :post, class: "btn pull-left subtab" %>
-        <% end %>
-      <% end %>
\ No newline at end of file
+<!-- profile and view buttons -->
+<% if signed_in? %>
+  <% if current_user == @user %>
+    <%= link_to raw('<i class="glyphicon glyphicon-edit"></i> Edit Your Profile'), sufia.edit_profile_path(@user), class: "btn pull-left subtab" %>
+  <% elsif current_user.following?(@user) %>
+    <%= link_to raw('<i class="glyphicon glyphicon-eye-close"></i> Unfollow'), sufia.unfollow_profile_path(@user), method: :post, class: "btn pull-left subtab" %>
+  <% else %>
+    <%= link_to raw('<i class="glyphicon glyphicon-eye-open"></i> Follow'), sufia.follow_profile_path(@user), method: :post, class: "btn pull-left subtab" %>
+  <% end %>
+  <%= link_to raw('<i class="glyphicon glyphicon-edit"></i> Change Your Display Name'), sufia.contact_path, class: "btn subtab" %>
+  <%= link_to raw('<i class="glyphicon glyphicon-globe"></i> View All Users'), sufia.profiles_path, class: "btn pull-right subtab" %>
+<% end %>

```

Comment:
Please only submit code changes. If you want to format source code, submit another PR.